ballerina/engine.bal

```diff
@@ -46,10 +48,24 @@ isolated class Engine {
         }
     }
 
-    isolated function execute(parser:OperationNode operationNode, Context context, map<Upload|Upload[]> fileInfo)
-        returns OutputObject {
-        ExecutorVisitor executor = new (self, self.schema, context, fileInfo);
-        OutputObject outputObject = executor.getExecutorResult(operationNode);
+    isolated function getResult(parser:OperationNode operationNode, Context context, map<Upload|Upload[]> fileInfo,
+        any result = ()) returns OutputObject {
```

Comment:
```suggestion
    isolated function getResult(parser:OperationNode operationNode, Context context, map<Upload|Upload[]> fileInfo,
                                any result = ()) returns OutputObject {
```

Reply:
This is the way the Ballerina style guide recommends using indentations. (There are places where we do not follow this, we should check them later).